cfn-resources/integration-tests/util/util.go

```diff
@@ -0,0 +1,56 @@
+// Copyright 2023 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//         http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package util
+
+import (
+	"crypto/rand"
+	"log"
+	"math/big"
+	"os"
+	"os/exec"
+)
+
+func GetRandNum() *big.Int {
+	randInt, _ := rand.Int(rand.Reader, big.NewInt(10000))
+	return randInt
+}
+
+func RunCleanupScript(rctx ResourceContext) {
+	output, err := exec.Command("/bin/sh", "../cleanup_cfn.sh").CombinedOutput()
+	if err != nil {
+		log.Println("Error when running cleanup script. Output:")
+		log.Println(string(output))
+		log.Printf("Got command status: %s\n", err.Error())
+		return
+	}
+	log.Printf("E2E test resource type %s successfully de-registered from private registry!\n", rctx.ResourceTypeNameForE2e)
+}
+
+func PublishToPrivateRegistry(rctx ResourceContext) {
+	os.Setenv("RESOURCE_TYPE_NAME", rctx.ResourceTypeName)
+	os.Setenv("RESOURCE_TYPE_NAME_FOR_E2E", rctx.ResourceTypeNameForE2e)
+	os.Setenv("E2E_RAND_SUFFIX", rctx.E2eRandSuffix)
+	os.Setenv("RESOURCE_DIRECTORY_NAME", rctx.ResourceDirectory)
```

Comment:
if this method is to only be used for tests let's make sure to use [func (*T) Setenv](https://pkg.go.dev/testing#T.Setenv) so the values are self cleaned at the end of the test run, also this is a meta comment that for test helpers it's ideal to pass the test variable and and call [func (*T) Helper](https://pkg.go.dev/testing#T.Helper) check https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/e2e/atlas/atlas_e2e_test_generator_test.go#L52

Reply:
updated